app/components/common/Modal.js

```diff
@@ -66,7 +66,7 @@ Modal.propTypes = {
   body: PropTypes.string,
   footer: PropTypes.string,
   close: PropTypes.func,
-  modal: PropTypes.object.isRequired
+  modal: PropTypes.object
```

Comment:
Just double checking: is this property not required anymore? It seems like the only purpose of this component is to display this modal.

Reply:
Same as above.